src/site/content/en/blog/webassembly-memory-debugging/index.md

```diff
@@ -0,0 +1,607 @@
+---
+title: Debugging memory leaks in WebAssembly
```

Comment:
```suggestion
title: Debugging memory leaks in WebAssembly using Emscripten
```

Last minute drive-by suggestion: I feel like we should mention “Emscripten” somewhere in the title as the article doesn’t really help you debug memory leaks when you are _not_ using Emscripten. Sound reasonable?

Reply:
Heh, nitially it said "...in WebAssembly wrappers" but I removed "wrappers" as the title felt a bit long. I guess we can put this in, but it makes it longer again.

Do you think this is an important piece of information?

Reply:
If it is, I wonder if at that point we should remove "WebAssembly" from the title altogether and make it "Debugging memory leaks with Emscripten".

Reply:
It’s a bigger thing I have been thinking about. TL;DR is that I think tooling is at the core of the WebAssembly story and I think it’s important that we mention in the title that this is Emscripten-centric. If you feel like my suggestion is too long, I am also totally okay with dropping “WebAssembly” from the title.

Reply:
> TL;DR is that I think tooling is at the core of the WebAssembly story and I think it’s important that we mention in the title that this is Emscripten-centric.

Totally agree with that reasoning. I guess my concern is that people who don't use Emscripten, will skip the article, even though the API design principles apply equally to any language, only the sanitizer part is Emscripten-specific.

Reply:
I personally don’t think that a non-Emscripten user will get a lot from this article. I totally see that the principles are transferrable, but without tooling support, that’s gonna be hard to do. So I’d vote for making this one Emscripten specific and link to (or write) articles for other tooling setups.

Reply:
Heh, perhaps adding sanitizers to the post wasn't a great idea after all. Initially this was meant to be a  language-agnostic post about safer API design for wrappers, but adding the complementary sanitizers part made it indeed very Emscripten-specific since only Emscripten implements the ASAN runtime for Wasm for now (this might change in future).

Fine, I'll rename the title, and will at least try to "fix" this by encouraging people to read it nevertheless in a tweet-announcement.